packages/cadl-typescript/package.json

```diff
@@ -39,11 +39,11 @@
     "ts-node": "^10.7.0",
     "typescript": "~4.8.0",
     "prettier": "~2.7.1",
-    "@azure-tools/cadl-ranch-specs": "~0.5.0",
+    "@azure-tools/cadl-ranch-specs": "0.5.0",
     "@cadl-lang/versioning": "0.10.0",
-    "@azure-tools/cadl-ranch-expect": "~0.1.12",
-    "@azure-tools/cadl-ranch": "~0.2.9",
-    "@azure-tools/cadl-dpg": "~0.3.0",
+    "@azure-tools/cadl-ranch-expect": "0.1.12",
+    "@azure-tools/cadl-ranch": "0.2.9",
+    "@azure-tools/cadl-dpg": "0.3.0",
```

Comment:
I am not upgrading the version here, just fix the version so that our test won't be affect by breaking changes in cadl ranch or cadl compiler.